servatrice/src/servatrice_database_interface.cpp

```diff
@@ -1091,3 +1091,23 @@ QList<ServerInfo_ChatMessage> Servatrice_DatabaseInterface::getMessageLogHistory
 
     return results;
 }
+
+bool Servatrice_DatabaseInterface::isUserADonator(const QString &user)
+{
+    if (!checkSql())
+        return false;
+    
+    QSqlQuery *query = prepareQuery("SELECT donator from {prefix}_users WHERE name = :user_name");
```

Comment:
That's not quite what I meant. I'm on my phone at the moment so it's hard
to type. I'll ping you on chat tonight and try to make my point more clear

On Tue, Oct 11, 2016, 11:41 AM woogerboy21 <notifications@github.com> wrote:

> In servatrice/src/servatrice_database_interface.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/2213#discussion_r82815196>:
>
> > @@ -1091,3 +1091,23 @@ QList<ServerInfo_ChatMessage> Servatrice_DatabaseInterface::getMessageLogHistory
> >
> >      return results;
> >  }
> > +
> > +bool Servatrice_DatabaseInterface::isUserADonator(const QString &user)
> > +{
> > +    if (!checkSql())
> > +        return false;
> > +
> > +    QSqlQuery *query = prepareQuery("SELECT donator from {prefix}_users WHERE name = :user_name");
>
> Sorry.... I see what your saying, expand out the ServerInfo_User
> information. Good idea. Though this will make the protocol even bigger as
> that info is passed back to the client for other purposes.
>
> @Daenyth <https://github.com/Daenyth>
> Do you still want this done?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/2213/files/7684d5f9590d28341e0fd0fb016aa7232411f81b#r82815196>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAA5NOJjvN8fuOk8y9NMBwWl_7msWLBmks5qy65AgaJpZM4KTNxT>
> .
>